src/ios/PushPlugin.m

```diff
@@ -119,8 +119,9 @@ - (void)unregister:(CDVInvokedUrlCommand*)command;
             [pubSub unsubscribeFromTopic:topic];
         }
     } else {
-        [[UIApplication sharedApplication] unregisterForRemoteNotifications];
```

Comment:
@ccorcos you can't delete this section as it is required for users who are using APNS to send messages. You have found an issue though and the code should check to see if `usesFCM` is true. If it is then you should call the `deleteIDWithHandler` method.